### PR TITLE
Update pre-remaster spell names

### PR DESCRIPTION
**Pull request by trowt**
Gentle Landing and Force Barrage had their pre-remaster names in the text.

### CODE REVIEW

File: packs/classfeatures/flexible-spell-preparation.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         },
         "category": "classfeature",
         "description": {
```
Comment (simonward) [inline packs/classfeatures/flexible-spell-preparation.json:13]: Typically we only link the first instance of a spell, so the renames shouldn’t be links.

Comment (trowt) [reply packs/classfeatures/flexible-spell-preparation.json:13]: Updated, thank you for the guidance

Comment (simonward) [reply packs/classfeatures/flexible-spell-preparation.json:13]: The non-link ,emotions should be lower-case to match the source text.

**Comment by simonward** [comment]
Looks like the source should be Secrets of Magic, not CRB.

File: packs/classfeatures/flexible-spell-preparation.json (hunk 1 above)
Comment (trowt) [reply packs/classfeatures/flexible-spell-preparation.json:13]: Shaved another yak.

**Comment by trowt** [comment]
> Looks like the source should be Secrets of Magic, not CRB.

This PR is to fix 2 spell names from Dungeons & Dragons 5th Edition. This is not a full review of the entire file. If I had added the source, fine, but this is unrelated and out of scope. I don't know enough to know what the source is, so I'm not going to change it. I do know that Magic Missile is spelled differently than Force Barrage.

If you don't want outside help, fine, say so. But the CONTRIBUTING.md makes it *seem* like help would be appreciated.

**Comment by stwlam** [comment]
I'll go ahead and merge this, but reconsider contributing in the future if you can't do so without losing your cool.

**Pull request merged**